### ISSUE TITLE
Murmur file name

### ISSUE THREAD

**Issue by KasperFranz** [wingify/vwo-php-sdk#7]
The namespace for murmur is wrong and causing this error on installation on composer 2:

```
Class vwo\Utils\Murmur located in ./vendor/vwo/vwo-php-sdk/src/Utils/murmur.php does not comply with psr-4 autoloading standard. Skipping.
```

**Comment by KasperFranz** [comment]
@softvar @griffinmarc 
any chance we can get this in :)?

**Comment by softvar** [comment]
@KasperFranz It's fixed in `v1.11.1`. Thanks for reporting this.
